database/schema.sql

```diff
@@ -0,0 +1,31 @@
+DROP DATABASE IF EXISTS impulse;
+
+CREATE DATABASE impulse;
+
+USE impulse;
+
+CREATE TABLE Users (
+  id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  username varchar(40) UNIQUE NOT NULL,
+  password varchar(64) NOT NULL,
+  salt varchar(64) NOT NULL
+);
+CREATE TABLE Sessions (
+  id int NOT NULL AUTO_INCREMENT,
```

Comment:
Why is Id not a PK value on this table? InnoDB is probably our MySQL engine and it prefers to have a PK value.